api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -51,50 +50,13 @@
 import org.hibernate.boot.registry.StandardServiceRegistry;
 import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
 import org.hibernate.cfg.Configuration;
-import org.junit.Before;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
-import org.openmrs.ConceptMapType;
+import org.openmrs.*;
 import org.openmrs.Order.Action;
-import org.openmrs.OrderAttribute;
```

Comment:
How is this related to switching from Hibernate mappings to annotations ,was it an overview?